doc/fd.1

```diff
@@ -405,5 +405,8 @@ $ fd -e py
 .TP
 .RI "Open all search results with vim:"
 $ fd pattern -X vim
+.TP
+.RI "Show ./ prefix in results by specifying search path:"
+$ fd pattern ./
```

Comment:
To be honest, I don't the we need this is an example. We should rather add the new `--strip-prefix` option in the normal overview of command line options.